draft-ietf-quic-transport.md

```diff
@@ -827,6 +848,24 @@ Implementations MUST assume that an unsupported version uses an unknown packet
 format. All other fields MUST be ignored when processing a packet that contains
 an unsupported version.
 
+## Latency Spin Bit
+
+The latency spin bit enables latency monitoring from observation points on
+the network path. This bit is set as follow:
+
+* The connection responder sets the spin bit value to the value of the
+  spin bit in the last packet received from the connection initiator.
+
+* The connection initiator sets the spin bit value to the opposite
```

Comment:
Ok, ekr explained to me out of band that I misunderstood how the spin bit worked.  I think the design here is more robust to loss than what I was thinking of, so I'm happy with it.

However, an overview sentence explaining what it's doing (ie: mark the entire flight with the spin bit, etc) would have prevented my confusion.